src/sqlfluff/utils/reflow/depthmap.py

```diff
@@ -28,10 +28,12 @@ def _stack_pos_interpreter(path_step: PathStep) -> str:
         # If there's only one code element, this must be it.
         elif len(path_step.code_idxs) == 1:
             return "solo"
-        # Check for whether first or last code element
-        elif path_step.idx == min(path_step.code_idxs):
+        # Check for whether first or last code element.
+        # NOTE: code_idxs is always sorted because of how it's constructed.
+        # That means the lowest is always as the start and the highest at the end.
+        elif path_step.idx == path_step.code_idxs[0]:
```

Comment:
Could this sortedness ever be broken in future refactoring? I'm not sure how to enforce this assumption forever.